examples/experimental/dagster-airlift/dagster_airlift/core/defs_from_airflow.py

```diff
@@ -17,11 +18,12 @@
 def build_defs_from_airflow_instance(
     airflow_instance: AirflowInstance,
     cache_polling_interval: int = DEFAULT_POLL_INTERVAL,
-    defs: Optional[Definitions] = None,
+    dag_defs_list: List[Union[DagDefs, Definitions]] = [],
```

Comment:
never default to `[]` https://docs.python-guide.org/writing/gotchas/

Reply:
Seems like we should force the caller to merge all the DagDefs into `Definitions` or _only_ take Dag Defs. This interemediate is not satisfying.